python/pyADflow.py

```diff
@@ -1682,6 +1692,146 @@ def Func(Xn, CLstar):
                 break
         return Xn
 
+    def solveTargetFuncs(self, aeroProblem, funcDict, tol=1e-4, nIter=10, Jac0=None):
+        """
+        Solve the an arbitrary set of function-dv sets using a Broyden method.
+
+        Parameters
+        ----------
+        AeroProblem : AeroProblem instance
+            The aerodynamic problem to be solved
+        funcDict : dict
+            Dictionary of function DV pairs to solve...add dv index {'func':{'dv':str, 'dvIdx':idx,'target':val,'initVal':val,'initStep':val}}
+        tol : float
```

Comment:
can you be a bit more clear at what this tolerance is? The current docstring is whats left from another function. We actually do the L2 norm of the error vector. So individual functions are not guaranteed to be solved to the tolerance. We may want to include an option to check for each function individually. Maybe scaling is bad on a function that does not converge very tightly, and a residual on the order of "tol" is not good enough, yet the norm is less than the norm due to other functions etc.

Reply:
I have updated the comment. I don't think this function is used in a way that makes individual scaling or error checking meaningful